qiskit_experiments/framework/composite/batch_experiment.py

```diff
@@ -65,6 +66,9 @@ def __init__(
                       provided this will be initialized automatically from the
                       supplied experiments.
         """
+        # Experiment identification metadata
+        self._type = experiment_type if experiment_type else type(self).__name__
```

Comment:
This will happen in the parent's constructor, what you have to do is to pass `experiment_type` to the parent's constructor.

By the way, in `BaseExperiment` you can modify this line to `self.experiment_type = experiment_type`.

Reply:
lol I should have read docs first, fixed it, removed `proparty` and `setter` from both `BaseExperiment` and `ParallelExperiment ` added init arg `experiment_type` in both.

Reply:
Remove this line.
Then, in `BaseExperiment`, replace this line with `self.experiment_type = experiment_type`.